themes/default/layouts/page/get-started.html

```diff
@@ -0,0 +1,53 @@
+{{ define "hero" }}
+    {{ partial "hero.html" (dict "title" .Params.title) }}
+{{ end }}
+
+{{ define "main" }}
+    <section class="container relative mx-auto mt-16 px-4 md:px-0">
+        <div class="shape-background get-started-install-section">
+            <div class="shape-container">
+                <div class="circle-right"></div>
+            </div>
+        </div>
+        <div class="mb-8 bg-violet-200 w-12 h-12 rounded-full mx-auto flex items-center justify-center">
+            <span class="text-violet-600 text-3xl font-display font-bold">1</span>
+        </div>
+        <h2 class="text-center">Install the Pulumi CLI</h2>
+        <p class="text-center">
+            <span class="max-w-md inline-block">To use the open source Pulumi SDK or the Pulumi Console, you'll need to install the Pulumi CLI.</span>
+        </p>
+        <div class="my-8 text-center">
+            <pulumi-install></pulumi-install>
+        </div>
+        <p class="text-center">Or review <a class="underline" href="{{ relref . "/docs/get-started/install" }}">other installation options</a>.</p>
+        <p class="text-center">Not ready to install yet? Watch a five-minute <a class="underline" href="https://www.youtube.com/watch?v=6f8KF6UGN7g" target="_blank" rel="noreferrer noopener">demo video</a> instead.</span></p>
+    </section>
+    <section class="container relative mx-auto mt-20 px-4 md:px-0">
+        <div class="shape-background get-started-step-2-section">
+            <div class="shape-container">
+                <div class="circle-left"></div>
+            </div>
+        </div>
+        <div class="mb-8 bg-salmon-200 w-12 h-12 rounded-full mx-auto flex items-center justify-center">
+            <span class="text-salmon-600 text-3xl font-display font-bold">2</span>
+        </div>
+        <h2 class="mb-8 text-center">Start learning or explore</h2>
+        <div class="lg:flex items-stretch mb-16">
+            <div class="w-full lg:w-1/2 px-4 py-2 lg:p-8 lg:px-4">
+                <div class="lg:shadow-2xl md:px-12 pb-12 pt-8 rounded-xl bg-white relative z-0">
+                    <h3><span class="text-2xl">Start learning</span></h4>
+                    <p class="mb-10">Pick a cloud and a language and deploy your first application with open source Pulumi!</p>
+                    <a href="{{ relref . "/docs/get-started" }}" class="btn btn-primary"><span class="hidden md:inline">Follow the </span>Getting Started Guide</a>
+                </div>
+            </div>
+
+            <div class="w-full lg:w-1/2 px-4 py-2 lg:p-8 lg:px-4">
+                <div class="lg:shadow-2xl md:px-12 pb-12 pt-8 rounded-xl bg-white">
+                    <h3><span class="text-2xl">Explore</span></h4>
+                    <p class="mb-10">Sign up to use the optional Pulumi Console, a UI where you can manage deployment state and collaborate with your team.</p>
```

Comment:
```suggestion
                    <p class="mb-10">Sign up to use the optional Pulumi Console, a UI where you can manage deployment state and collaborate with your team!</p>
```

Reply:
FWIW, I kinda liked your earlier revision to use a period here. It had a calmer feel to it.